src/kw_include.sh

```diff
@@ -0,0 +1,31 @@
+#Used for file sourcing.
+#This function should used for file sourcing instead of `. file.sh --source-only`
+#
+# @filepath Path to the file to be sourced
+#
+# Returns:
+# 0 on a succesful import, 1 if the file can't be read and 2 if it can´t be found
+function include() {
+
+  local filepath="$1"
+  local varname=$(basename "$filepath" .sh)
+
+  if [[ ! -e "$filepath" ]]; then
+    echo "File $filepath could not be found, check your file path."
+    return 2 # ENOENT
+  fi
+
+  if [[ ! -r "$filepath" ]]; then
+    echo "File $filepath could not be read, check your file permissions."
+    return 1 # EPERM
+  fi
+
+  varname=${varname^^}_IMPORTED # capitalize and append "_IMPORTED"
+
+  if [[ -v "${varname}" ]]; then
+    return 0
+  fi
+
+  declare -g "${varname}"=1
+  . "$filepath" --source-only > /dev/null
```

Comment:
I added a `> /dev/null` because the kw_test was including kw and messing up the tests output:

```
Running test [kw_test]
=========================================================
Invalid option
Usage: kw [options]

The current supported targets are:
         Host - this machine
         Qemu - qemu machine
         Remote - machine reachable via the network

Commands:
        bd - Build and install modules
        build|b - Build kernel
        clear-cache - Clear files generated by kw
        codestyle,c - Apply checkpatch on directory or file
        configm,g - Manage config files
        deploy,d - Deploy a new kernel image to a target machine
        diff,df - Diff files
        drm - Set of commands to work with DRM drivers 
        explore,e - Explore string patterns
        help,h - displays this help mesage
        init - Initialize kworkflow config file
        maintainers,m - Get maintainers and mailing list
        man - Show manual
        mount,mo - Mount partition with qemu-nbd
        pomodoro,p - kw pomodoro support
        ssh,s - SSH support
        statistics - Provide basic statistics related to daily development
        umount,um - Umount partition created with qemu-nbd
        up,u - Wake up vm
        vars,v - Show variables
        version,--version,-v - show kw version

validate_global_variables_Test
check_kworkflow_global_variable_Test
set_KW_LIB_DIR_in_dev_mode_Test

Ran 3 tests.
 ```